org.eclipse.scanning.event/src/org/eclipse/scanning/event/AbstractQueueConnection.java

```diff
@@ -531,39 +537,39 @@ public long getMaximumCompleteAge() {
 		return A_WEEK;
 	}
 
-	
+
 	public void pause() throws EventException {
 		throw new EventException("Method 'pause' is not implemented!");
 	}
-	
+
 	public void resume() throws EventException {
 		throw new EventException("Method 'pause' is not implemented!");
 	}
 
 	public boolean isQueuePaused(String submissionQueueName) {
-		
+
 		PauseBean bean = getPauseBean(submissionQueueName);
 		return bean!=null ? bean.isPause() : false;
 	}
-	
+
 	private PauseBean getPauseBean(String submissionQueueName) {
-		
+
 		IQueueReader<PauseBean>   qr=null;
 		try {
 			qr = eservice.createQueueReader(getUri(), EventConstants.CMD_SET);
 			qr.setBeanClass(PauseBean.class);
 		    List<PauseBean> pausedList = qr.getQueue();
-		    
+
 		    // The most recent bean in the queue is the latest
 		    for (PauseBean pauseBean : pausedList) {
 				if (submissionQueueName.equals(pauseBean.getQueueName())) return pauseBean;
 			}
-		    
+
 		} catch (Exception ne) {
 			ne.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1148)